src/presets/render.ts

```diff
@@ -0,0 +1,5 @@
+import { defineNitroPreset } from '../preset'
+
+export const render = defineNitroPreset({
```

Comment:
Calling it `render` alone, is a little bit confusing for someone who doesn't know their platform... (although smart marketing move!). Can we use `rendercom` or `onerender` maybe?

Reply:
`render-com`/`renderCom`?